bin/review-rot

```diff
@@ -62,13 +64,16 @@ def main(state_, value, duration, ssl_verify=None):
                 """
                 res = format_user_repo_name(data)
                 """
-                get pull requests for specified git service, username
-                and repository name
+                get pull/merge/change requests for specified git service
                 """
+                repo_name = data
```

Comment:
Method "format_user_repo_name" returns dict which has "repo_name" key with value as None.
So res.get('repo_name', data) will give None when username is not present.
Modified method "format_user_repo_name" to handle this.